src/ExtranetBundle/LoginByTransferToken/TransferTokenService.php

```diff
@@ -92,6 +105,21 @@ private function decodeToken(string $token): ?array
         return $data;
     }
 
+    private function isSecretIsDefaultFromQuickstartTemplate(): bool
```

Comment:
I find it kindof irritating to repeat the "is" in the method name. Wouldn't it be easier to read `isSecretDefaultFromQuickstartTemplate`?

Reply:
That's what happens when you are unsure which naming scheme you want to use: You use both at the same time.

:heavy_check_mark: Fixed by renaming the method to `isReadyToEncodeTokens` which also resolves the issue of duplication